lib/ord.js

```diff
@@ -176,68 +132,68 @@ const fGetEventResources = (csn, global, packageIds) => {
     }
 
     return services.map((serviceName) =>
-        fCreateEventResourceTemplate(
+        createEventResourceTemplate(
             serviceName,
             csn.definitions[serviceName],
-            global,
+            appConfig,
             packageIds
         )
     );
 };
 
 module.exports = (csn) => {
     const linkedCsn = cds.linked(csn);
-    Object.assign(global, fInitializeGlobal(linkedCsn));
```

Comment:
The "problematic" flow would be:
1. We define appConfig and use appConfig in some function as an input parameter
2. We change property e.g. aServices in one of the functions called in the sequence
3. We use appConfig in some of the next functions in the sequence where we again check aServices for some criteria which is assuming that appConfig.aServices has the original value, BUT we filltered-out some values in the step 2
I'm saying it is the case right now, I'm stating it is a bad practice and makes code harder for maintenance because you don't know if in the future there will be some new functions added that are using appConfig and assuming it is unchanged...

Reply:
Thanks, I get your point, but I'm not clear why do we need to assume or consider the appConfig has the original value? I think this question is key whether we should use cloned value. And I also want to know why it's bad practice? In some functions, I also use cloned [object](https://github.com/cap-js/ord/blob/main/lib/extendOrdWithCustom.js#L24). From my side, cloned object is expensive. We should use it when it really needed.